### DISCUSSION TITLE
Future of this Project.

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#1018] [General]
Hey,

This project was last updated in December, 2022 (almost 3 months ago), do you plan on making future releases, etc @ThioJoe or are you done with this project?

**Comment by ThioJoe** [comment]
I still plan to maintain it. I actually need to make a new release that I've had pending for a while.

**Reply by CasonPlayzYT** [reply]
Sounds good

**Comment by CasonPlayzYT** [comment]
sounds good
